src/screens/FeedScreen/hooks/useCoreFeed.js

```diff
@@ -4,21 +4,15 @@ import axios from 'axios';
 import {useSafeAreaInsets} from 'react-native-safe-area-context';
 
 import StorageUtils from '../../../utils/storage';
-import dimen from '../../../utils/dimen';
+import useViewPostTimeHook from './useViewPostTimeHook';
 import {Context} from '../../../context';
 import {FEEDS_CACHE} from '../../../utils/cache/constant';
-import {SOURCE_FEED_TAB} from '../../../utils/constants';
 import {checkIsHasColor, hexToRgb} from '../../../utils/colors';
 import {downVote, upVote} from '../../../service/vote';
-import {getFeedDetail, getMainFeedV2WithTargetFeed, viewTimePost} from '../../../service/post';
+import {getFeedDetail, getMainFeedV2WithTargetFeed} from '../../../service/post';
 import {listFeedColor} from '../../../configs/FeedColor';
 import {saveToCache} from '../../../utils/cache';
-import {
-  setFeedByIndex,
-  setMainFeeds,
-  setTimer,
-  setViewPostTimeIndex
-} from '../../../context/actions/feeds';
+import {setFeedByIndex, setMainFeeds, setTimer} from '../../../context/actions/feeds';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statements have been updated to include the new `useViewPostTimeHook` and remove unused imports such as `dimen` and `viewTimePost`. This is a good practice for maintainability and readability of the code.
```diff
- import dimen from '../../../utils/dimen';
+ import useViewPostTimeHook from './useViewPostTimeHook';
- import {getFeedDetail, getMainFeedV2WithTargetFeed, viewTimePost} from '../../../service/post';
+ import {getFeedDetail, getMainFeedV2WithTargetFeed} from '../../../service/post';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->